docs/mmteb/points.md

```diff
@@ -15,4 +20,9 @@ First name and last name are as you want them to appear in a publication.
 | ----------------- | ---------- | ---------- | ---------------------------- | -------------------- |
 | KennethEnevoldsen | Kenneth    | Enevoldsen | kennethcenevoldsen@gmail.com | ~Kenneth_Enevoldsen1 |
 | x-tabdeveloping   | Márton     | Kardos     |                              | ~Márton_Kardos1      |
+| imenelydiaker     | Imene      | Kerboua    |                              |                      |
+| wissam-sib        | Wissam     | Siblini    |                              |                      |
+| GabrielSequeira   | Gabriel    | Sequeira   |                              |                      |
+| schmarion         | Marion     | Schaeffer  |                              |                      |
+| MathieuCiancone   | Mathieu    | Ciancone   |                              |                      |
```

Comment:
Unfortunately we don't all have OpenReview accounts. Personnally, I will be able to add mine in 2 months approximatively (still have paperwork to do with my university). I think only @schmarion has one in our team, @wissam-sib do you have one?

Reply:
```suggestion
| imenelydiaker     | Imene      | Kerboua    |                              |                      |
| wissam-sib        | Wissam     | Siblini    |                              |                      |
| GabrielSequeira   | Gabriel    | Sequeira   |                              |                      |
| schmarion         | Marion     | Schaeffer  |                              |  ~Marion_Schaeffer1 |
| MathieuCiancone   | Mathieu    | Ciancone   |                              |                      |
```

Reply:
I believe it required for publication (at least at ACL ARR):

> All authors must be registered in OpenReview. Your OpenReview profile must be up to date (including affiliations and a recent import of any DBLP-listed publications) to aid in the detection of conflicts of interest.

Should we also add affiliation here as well (would be problematic to collect it later on).

Reply:
Actually for ACL it's not required for all authors. Only one author needs an account. It's just that authors with no OpenReview accounts won't be able to access the ARR cycle (comment on reviews, etc.)

For the affiliations I'll add a column then?

Reply:
> Actually for ACL it's not required for all authors. Only one author needs an account. It's just that authors with no OpenReview accounts won't be able to access the ARR cycle (comment on reviews, etc.)

Really! So ARR is different. Probably good to get it added though in case we go for ARR, but feel free to add it later.

> For the affiliations I'll add a column then?

That would be lovely.